pxd.c

```diff
@@ -1465,13 +1457,6 @@ ssize_t pxd_export(struct fuse_conn *fc, uint64_t dev_id)
 	spin_lock(&pxd_dev->lock);
 	pxd_dev->exported = true;
 	spin_unlock(&pxd_dev->lock);
-#if defined __PX_BLKMQ__ && !defined __PXD_BIO_MAKEREQ__
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(6,14,0) || defined(__EL9_STREAM__)
-	blk_mq_unfreeze_queue(pxd_dev->disk->queue, blk_mq_queue_flag);
-#else
-	blk_mq_unfreeze_queue(pxd_dev->disk->queue);
-#endif
-#endif
```

Comment:
same here, how can this PR consume `blk_mq_queue_flag` when it is not part of the code? seems it was already removed?